packages/language-server/src/server.ts

```diff
@@ -210,19 +207,15 @@ export function startServer(options?: LSOptions) {
                 colorProvider: true,
                 documentSymbolProvider: true,
                 definitionProvider: true,
-                codeActionProvider: clientCodeActionCapabilities?.codeActionLiteralSupport
+                codeActionProvider: evt.capabilities.textDocument?.codeAction
+                    ?.codeActionLiteralSupport
                     ? {
                           codeActionKinds: [
                               CodeActionKind.QuickFix,
                               CodeActionKind.SourceOrganizeImports,
                               SORT_IMPORT_CODE_ACTION_KIND,
                               ...(clientSupportApplyEditCommand ? [CodeActionKind.Refactor] : [])
-                          ].filter(
-                              clientSupportedCodeActionKinds &&
-                                  evt.initializationOptions.shouldFilterCodeActionKind
-                                  ? (kind) => clientSupportedCodeActionKinds.includes(kind)
-                                  : () => true
-                          )
+                          ]
```

Comment:
the above changes seem to be an accidental revert of some changes that went into master

Reply:
good catch @dummdidumm , I've pushed an update to bring it up to latest